UI/css/style.css

```diff
@@ -63,6 +67,17 @@ body{
 	color: var(--dark);
 }
```

Comment:
@Alexander96779 Add a height on the wrapper of `100vh`. This will fix the problem of your work showing halfway the page but again it will increase the size of the header and footer because the footer is inside the wrapper. So also go ahead a provide height for both the footer and header.

![image](https://user-images.githubusercontent.com/33352484/69685900-ae2d6100-10c6-11ea-9a93-1f14a0cd1c8a.png)